src/components/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import Header from '../elements/Header';
+import './Main.scss';
+
+const Main = () => {
+  const headerText = 'Приемы';
+
+  return (
+    <div>
+      <Header
+        headerText={headerText}
+        children={<Button className="header-btn">Выйти</Button>}
```

Comment:
please add correct children

Reply:
this button will be correctly in next pull requests, when i can test his with `Login` functionally